src/garrysmod/Dockerfile

```diff
@@ -8,3 +8,30 @@ RUN set -ex \
     +quit
 
 COPY cfg/mount.cfg garrysmod/cfg/mount.cfg
+
+# Switch to x64 branch
+RUN set -ex \
+    && /usr/games/steamcmd +login anonymous \
+    +app_update 4020 \
+    -beta x86-64 \
+    validate \
+    +quit
+
+USER root
+
+ENV LIBPQ_VERSION 11.7-0+deb10u1
+ENV LIBPQXX_VERSION 6.2.5-1
```

Comment:
These match the versions I am compiling gmsv_pg against.

https://github.com/rbreslow/gmsv_pg/blob/0b7c3e3f377acf51d99ca51284ce1cedff4a7fcd/Dockerfile#L5-L6